apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -1,11 +1,123 @@
+.btn {
+  color: white;
+  text-transform: uppercase;
+  text-align: center;
+  width: 100%;
+  background-color: #D32E6A;
+  padding: 0.5em;
+  cursor: pointer;
+}
+
+.close-btn {
+  position: absolute;
+  top: -6.4rem;
+  right: 0;
+  color: #D32E6A;
+  font-size: 8rem;
+  cursor: pointer;
+}
+
+.main-column {
+  width: calc(67% - 10px);
+  margin: 5px;
+}
+
+.side-column {
+  width: calc(33% - 10px);
+  margin: 5px;
+
+  img {
+    max-width: 100%;
+  }
+}
+
 .reactions {
+  .reactions-modal {
+    display: none;
+  }
+
+  &.open {
+    .reactions-modal {
+      display: block;
+      z-index: 10000;
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+
+      &:before {
+        position: fixed;
+        display: block;
+        content: ' ';
+        width: 100%;
+        height: 100%;
+        background-color: white;
+        opacity: 0.93;
+      }
+    }
+
+    .reactions-container {
+      display: flex;
+      flex-flow: row wrap;
+      margin: -5px;
+      position: absolute;
+      top: 10%;
+      left: 50%;
+      transform: translateX(-50%);
+      width: 90%;
+      max-width: 1000px;
+    }
+  }
+}
+
+.reaction-list {
+  display: flex;
   width: 100%;
+  justify-content: space-between;
+  flex-flow: row wrap;
+}
+
+.reaction-list-item {
+  position: relative;
+  width: 100%;
+  margin-bottom: 4%;
+
+  @media (min-width: 650px) {
+    width: 47%;
+  }
+
+  @media (min-width: 950px) {
+    width: 30%;
+  }
+
+  @media (min-width: 1400px) {
+    width: 22%;
+  }
+
+  @media (min-width: 1900px) {
+    width: 18%;
+  }
+
+  &:after {
+    content: ' ';
+    padding-bottom: 120%;
+    display: block;
+  }
+}
+
+.reaction-list {
```

Comment:
Merge rule `.reaction-list` with rule on line 74

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#mergeableselector)